.github/workflows/flake8.yml

```diff
@@ -1,5 +1,8 @@
 name: Flake8
-on: pull_request
+on: 
+  pull_request:
+    branches:
+    - main
```

Comment:
Explicitly setting this filters the events from this specific branch.
It may be fine to do it, just reason for it in the commit.

Just note that this is not a must here and it will fit as an independent PR, it has little to do with fixing the current flake8 errors.

Reply:
It is not a must because everything will still work without it.

Reply:
> It is not a must because everything will still work without it.

for me, it didn't work until I changed the file.

Reply:
> Explicitly setting this filters the events from this specific branch.
> It may be fine to do it, just reason for it in the commit.
> 
> Just note that this is not a must here and it will fit as an independent PR, it has little to do with fixing the current flake8 errors.

I don't fully understand what you mean, just move the changes of the py files to another PR?

Reply:
> > It is not a must because everything will still work without it.
> 
> for me, it didn't work until I changed the file.

I tried and it worked: https://github.com/beyond-io/picATrip/pull/29

Force push without it and lets see.